index.js

```diff
@@ -35,14 +35,14 @@ ShareDialog.prototype.get = function() {
       s = decodeURIComponent(s)
     return encodeURIComponent(s)
   })
-  return this._url + '?' + params
+  return this._url + ((params!='') ? '?' + params : '');
```

Comment:
For readability, I suggest to lose the parentheses and put some distance between the question marks:

```js
return params ? this._url + '?' + params : this._url
```